apps/server/api/views.py

```diff
@@ -30,12 +34,73 @@ def field_value_object_key(serializer: serializers.Serializer) -> Optional[str]:
     return object_key
 
 
+def create_composite_tabular_data_file(
+        experiment_name: str,
+        experiment_settings: list,
+        location_tags: dict
+        ) -> str:
+
+    composite_tabular_data_file_name = f"{experiment_name}/composite_tabular_data.csv"
```

Comment:
We want to create a parquet file for the combined file right?

Also, I would suggest using pandas for this. That _should_ make the code relatively simple. Load X data frames from the CSV file. Combine them into one data frame vertically, then save `to_parquet`.

Reply:
Okay, we can make that change. But yes, originally was parquet and I completely forgot when I was trying to get this thing set up. Shouldn't be much of a hiccup.